Add CrUX July 2022 changelog (#3418)
GoogleChrome/developer.chrome.com: site/en/docs/crux/release-notes/index.md

In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.

## 202207

[Announcement](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/N5B3F9QVOmY)

Publication date
 : August 09, 2022

What's new

>  : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. See the [CLS changelog](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/aJ4ni2LNVsU) for details.

Reviewer: ```suggestion
 : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```

Author: Oh good spot! Copied the wrong URL as didn't include the docs announcement in here.

>  : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. Please see the details in the [CLS changelog](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/aJ4ni2LNVsU).

Reviewer: ```suggestion
 : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. Please see the details in the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md).
```

>  : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.

Reviewer 2: ```suggestion
 : - This release includes a change which records the CLS metric at the first `OnHidden` in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```

Reviewer 2: This _seems_ like it should be in a code font, but reject this if not.

Author: Let's reword it instead, as not sure how much of that is an internal status rather than a web standard one.

```suggestion
 : - This release includes a change which records the CLS metric when the tab is bacgrounded in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```
